### PR TITLE
more details for examples script container

### PR DESCRIPTION
**Pull request by mikeblas**

### CODE REVIEW

**Review by dsmmcken — APPROVED** [review]
That helped, thanks.

File: README.md [hunk 1]
```diff
@@ -27,4 +27,14 @@ Each folder in this repository has the following structure within:
    1. `docker build -t deephaven/examples samples`
    2. `docker run --rm -v "$(pwd)/docker/core/data:/data" deephaven/examples download`
 
-The commands above mount `docker/core/data/examples` in the `deephaven-core` clone as `/data/examples` within the docker container.
+The `docker build` command builds a Docker container which hosts a script that helps manage the examples. That script will automate the management of the example files, allowing you to download and update the example files without directly working with the git project hosting the examples. This command is only needed once; after it runs, the container is available to use on your host.
```
Comment (chipkent) [inline README.md:30]:
```suggestion
The `docker build` command builds a Docker container containing a script that helps manage the examples. 
 That script will automate the management of the example files.  It allows you to download and update the example files without directly working with the git project hosting the examples.  This `docker build` command is only needed once; after it runs, the container is available to use on your host.
```

File: README.md [hunk 2]
```diff
@@ -27,4 +27,14 @@ Each folder in this repository has the following structure within:
...
+
+The `docker run` command runs that new container, letting it know that the examples will end up in your `deephaven-core/docker/core/data` path. When Deephaven runs in its containers, this directory from the host is mounted inside the container as `/data/examples`.
```
Comment (chipkent) [inline README.md:32]:
```suggestion
The `docker run` command downloads the example data by running the new examples management container.  The `-v "$(pwd)/docker/core/data:/data"` argument mounts your local `deephaven-core/docker/core/data` path as `/data` in the container.  The example data is stored to `/data/examples` inside the container, which is `deephaven-core/docker/core/data/examples` on the local filesystem.
```

File: README.md [hunk 3]
```diff
@@ -27,4 +27,14 @@ Each folder in this repository has the following structure within:
...
+
+You can run `docker run` again to manage the example data -- for example, to download a new version, or remove the sample data. This command:
+
+	`docker run --rm -v "$(pwd)/docker/core/data:/data" deephaven/examples`
+
+will run the contained script and show usage for the different commands it offers.
```
Comment (chipkent) [inline README.md:38]:
```suggestion
You can run `docker run` again to manage the example data -- for example, to download a new version. To see what options are available, run:

	`docker run --rm -v "$(pwd)/docker/core/data:/data" deephaven/examples`
```

**Review by chipkent — CHANGES_REQUESTED** [review]

**Review by chipkent — APPROVED** [review]

**Review by chipkent — APPROVED** [review]

**Pull request merged**